Source/API/CBLManager.m

```diff
@@ -726,10 +726,26 @@ - (void) _forgetDatabase: (CBLDatabase*)db {
         return [repl localDatabase] == db;
     }];
     [_databases removeObjectForKey: name];
-    [_shared closedDatabase: name];
+
+    CBL_Shared* strongShared = _shared;
+    [strongShared closedDatabase: name];
+
+    // Close background database:
+    [strongShared.backgroundServer tellDatabaseManager: ^(CBLManager* bgmgr) {
+        NSError* error;
+        if (![bgmgr _closeDatabaseNamed: name error: &error])
+            Warn(@"Cannot close background database named %@: %@", name, error);
+    }];
 }
 
 
+- (BOOL) _closeDatabaseNamed: (NSString*)name error: (NSError**)outError {
+    CBLDatabase* db = _databases[name];
+    if (db)
```

Comment:
This doesn't initialize `*outError` if db is nil. It should at least set it to nil.